Automattic/jetpack
# E2E tests: run all relevant e2e tests on updates in mirror repos

#### Changes proposed in this Pull Request:

Follow-up on #27384, run all relevant e2e tests when repository dispatch event is received for updates on mirror repos.
- Replace the test plan creation script with a new, nodejs one. It's easier to read and work with.
- Evaluate the mirror repo name against targets defined in e2e projects package.json and decide what to run

#### Other information:


#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* On pull_request event nothing should change. Check that all relevant tests ran for this PR.
* On push all tests should run. Tested the script locally. Entire workflow to be tested after merge.
* On repository_dispatch only tests relevant to the repo that sent the dispatch should run. Tested the script locally. Entire workflow to be tested after merge.
